python/tank/platform/software_launcher.py

```diff
@@ -495,16 +501,24 @@ def get_standard_plugin_environment(self):
         # note: get_shotgun_id() returns None for unmanaged configs.
         pipeline_config_id = self.sgtk.pipeline_configuration.get_shotgun_id()
         if pipeline_config_id:
-            self.logger.debug(
-                "Setting SHOTGUN_PIPELINE_CONFIGURATION_ID to %s", pipeline_config_id
```

Comment:
On previous comment on this PR suggested that I add some logging when setting `SHOTGUN_BUNDLE_CACHE_FALLBACK_PATHS`. Due to Github hiding comments on line that were edited, we're now missing the response.

I argued that since we are already logging the environment at the bottom of the method that we didn't need to log anything when we actually set the value.